src/django_scrypt_fmt_plug.c

```diff
@@ -188,27 +192,25 @@ static int crypt_all(int *pcount, struct db_salt *salt)
 
 #ifdef _OPENMP
 #pragma omp parallel for
-	for (index = 0; index < count; index++)
 #endif
-	{
+	for (index = 0; index < count; index++) {
 		if (crypto_scrypt((unsigned char*)saved_key[index], strlen((char*)saved_key[index]),
 				cur_salt->salt, strlen((char*)cur_salt->salt),
 				(1ULL) << cur_salt->N, cur_salt->r,
 				cur_salt->p, (unsigned char*)crypt_out[index],
-				BINARY_SIZE) == -1)
-		{
+				BINARY_SIZE) == -1) {
 			memset(crypt_out[index], 0, sizeof(crypt_out[index]));
```

Comment:
I think we shouldn't silently ignore errors from scrypt. They are not same as a wrong password. It'd be more appropriate to print a message and maybe even `error()` out afterwards. But this is not a cleanup matter, so should be in a separate PR, and perhaps not limited to this one instance of the problem.

Reply:
Ack.